Icon deferring; match all SVG tags

royduin opened the pull request:
- Initially created with a few common tags: https://github.com/blade-ui-kit/blade-icons/pull/191/files#diff-c17f4a19b6b187c3f1e3c05b299f088d65696dc40479b19d3dc96197f44c05c5R50
- Later on it's extended with: https://github.com/blade-ui-kit/blade-icons/pull/200/files#diff-c17f4a19b6b187c3f1e3c05b299f088d65696dc40479b19d3dc96197f44c05c5R50
- And even more tags: https://github.com/blade-ui-kit/blade-icons/pull/202/files#diff-c17f4a19b6b187c3f1e3c05b299f088d65696dc40479b19d3dc96197f44c05c5R50
- But there are way more: https://developer.mozilla.org/en-US/docs/Web/SVG/Element

With this PR it doesn't matter anymore, every tag works.

royduin committed:
Icon deferring; match all SVG tags

royduin commented:
Should we still support Laravel 8? EOL for a while now...

royduin commented:
cc @indykoning  @pionl @AEM5299

driesvints commented on src/Svg.php:
```diff
@@ -50,7 +50,10 @@ protected function deferContent(string $contents, $defer = false): string
+        $svgContent = str($contents)
+            ->replaceMatches('/<svg[^>]*>/', '')
+            ->replaceMatches('/<\/svg>/', '')
+            ->toString();
```
Can we reference the `Str` class here instead of the helper?

driesvints commented:
@royduin no we can remove it but can we do that in a separate PR? Can you send one in? Thanks 🙏

royduin committed:
Don't use the helper

royduin committed:
Use __toString()

royduin commented:
Changed it to class instead of the helper, also replaced `toString()` with `__toString()` which also makes it Laravel 8 compatible. No need to drop Laravel 8 support, yet.

driesvints commented:
Awesome, thanks @royduin
